code/modules/reqs/supplypacks.dm

```diff
@@ -838,11 +838,21 @@ WEAPONS
 	contains = list(/obj/item/ammo_magazine/pistol/standard_pistol/smart_pistol)
 	cost = 10
 
-/datum/supply_packs/weapons/ltb_shells
-	name = "LTB tank shell"
+/datum/supply_packs/weapons/ltb_he_shell
+	name = "LTB High Explosive tank shell"
 	contains = list(/obj/item/ammo_magazine/tank/ltb_cannon)
 	cost = 10
```

Comment:
```suggestion
	cost = 75
```
This and APFDS. Cost more considering they nuke people on hit. This suggested price is 15 more than SADAR AP for reference.